cluster/examples/kubernetes/ceph/object-zone.yaml

```diff
@@ -0,0 +1,14 @@
+#################################################################################################################
+# Create a multisite zone for a collection of object stores
+#  kubectl create -f object-zone.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStoreZone
+metadata:
+  name: my-zone
+  namespace: rook-ceph
+spec:
+  realm: my-realm
+  zoneGroup: my-zonegroup
+  isMaster: true
```

Comment:
I thought we decided the `isMaster` flag wasn't needed, and the first zone or zonegroup would just be made the master if no others exist yet. Or am I forgetting?